test/integration/test/captions.test.js

```diff
@@ -1,12 +1,24 @@
 const clientHelper = require('../src/client-helper');
 const configHelper = require('../src/config-helper');
+const InstanceStateError = require('../../../src/models/RevAiApiError').InvalidStateError;
 
 test('Can get srt captions', async (done) => {
     const client = clientHelper.getClient(configHelper.getApiKey());
     const jobList = await client.getListOfJobs();
     const jobId = clientHelper.getTranscribedJobId(jobList);
     expect(jobId).toBeDefined();
-    const captionsStream = await client.getCaptions(jobId);
+    var captionStream = null;
+    try {
+        captionsStream = await client.getCaptions(jobId);
+    }
+    catch (error){
```

Comment:
```suggestion
    catch (error) {
```

Reply:
:0 I'm surprised the linter didn't catch this. Will fix